src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -1126,6 +1133,11 @@ public function move($blnIsAjax=false)
 			// Redirect or reload
 			if (!$objUploader->hasError())
 			{
+				if ($blnIsAjax)
+				{
+					throw new ResponseException(new Response(\Message::generateUnwrapped(), 201));
```

Comment:
This is the wrong status code. If the response has content, it should be `200`, otherwise `204`.